src/ReCaptchaComponent.tsx

```diff
@@ -19,18 +20,18 @@ const patchPostMessageJsCode = `(${String(function () {
   window.postMessage = patchedPostMessage
 })})();`
 
-const getExecutionFunction = (siteKey: string) => {
-  return `window.grecaptcha.execute('${siteKey}', { action: 'login' }).then(
+const getExecutionFunction = (siteKey: string, action: string) => {
+  return `window.grecaptcha.execute('${siteKey}', { action: '${action}' }).then(
```

Comment:
just a nitpick, but there's no need for a template string when it just contains a single value - this is equivalent:

```suggestion
  return `window.grecaptcha.execute('${siteKey}', { action }).then(
```